app/views/workflows/index.html.erb

```diff
@@ -32,29 +32,47 @@ See doc/COPYRIGHT.rdoc for more details.
   <p class="nodata"><%= l(:label_no_data) %></p>
 <% else %>
   <div class="autoscroll">
-    <table class="list">
-      <thead>
-        <tr>
-          <th></th>
-          <% @workflow_counts.first.last.each do |role, count| %>
-            <th>
-              <%= content_tag(role.builtin? ? 'em' : 'span', h(role.name)) %>
-            </th>
-          <% end %>
-        </tr>
-      </thead>
-      <tbody>
-        <% @workflow_counts.each do |type, roles| -%>
-          <tr class="<%= cycle('odd', 'even') %>">
-            <td><%= h type %></td>
-            <% roles.each do |role, count| -%>
-              <td align="center">
-                <%= link_to((count > 0 ? count : image_tag('webalys/red_x.png')), {:action => 'edit', :role_id => role, :type_id => type}, :title => l(:button_edit)) %>
-              </td>
+    <div class="generic-table--container">
+      <div class="generic-table--results-container">
+        <table interactive-table role="grid" class="generic-table">
+          <colgroup>
+            <col>
+            <col highlight-col>
+            <col highlight-col>
+            <col highlight-col>
+            <col highlight-col>
+            <col highlight-col>
+            <col>
+          </colgroup>
+          <thead>
+            <tr>
+              <th></th>
+              <% @workflow_counts.first.last.each do |role, count| %>
+                <th>
+                  <div class="generic-table--sort-header-outer">
+                    <div class="generic-table--sort-header">
+                        <%= content_tag(role.builtin? ? 'em' : 'span', h(role.name)) %>
+                    </div>
+                  </div>
+                </th>
+              <% end %>
+            </tr>
+          </thead>
+          <tbody>
+            <% @workflow_counts.each do |type, roles| -%>
+              <tr>
+                <td><%= h type %></td>
+                <% roles.each do |role, count| -%>
+                  <td>
+                    <%= link_to((count > 0 ? count : image_tag('webalys/red_x.png')), {:action => 'edit', :role_id => role, :type_id => type}, :title => l(:button_edit)) %>
```

Comment:
Not directly related to the PR (maybe do it in a separate one) but can we get rid of `image_tag('webalys/red_x.png')`. That way we can sever the connection to app/assets/images/webalys even more. 

I created a work package for it: https://community.openproject.org/work_packages/21312